### ISSUE TITLE
Updating a property name on the schema keeps the old property name

### ISSUE THREAD

**Issue by PaulKinlan** [breadboard-ai/breadboard#930]
I updated the `text` name and id to be `article`, it created the new one (article), but left the old one around (text)

<img width="651" alt="Screenshot 2024-03-01 at 12 18 13" src="https://github.com/breadboard-ai/breadboard/assets/45510/4dd418cf-e66f-4fa6-b0de-a0589b0eb82c">

**Comment by paullewis** [comment]
Yeah, that's an interesting one. It happens because we support ports by virtue of you declaring an edge to exist. So if you have an edge from `input-1.text` to `output-2.text` and you rename the port the edge still exists, and so Breadboard decides to create an "ephemeral port" for `text`. So it's not broken per se, but it's also unexpected.

One option is that we cascade delete all the edges from that port, and another is that we try to optimistically rename them... or we leave it as-is.

**Comment by paullewis** [comment]
Closing as obsolete
